recipes/msix/all/conanfile.py

```diff
@@ -0,0 +1,140 @@
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "crypto_lib": "openssl",
+        "pack": False,
+        "skip_bundles": False,
+        "use_external_zlib": True,
+        "use_validation_parser": False,
+        "xml_parser": "msxml6"
+    }
+
+    generators = "cmake"
+    exports_sources = "CMakeLists.txt", "patches/**"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        if self.settings.os == "Android":
+            self._cmake.definitions["AOSP"] = True
+        if self.settings.os == "Linux":
+            self._cmake.definitions["LINUX"] = True
+        if self.settings.os == "Macos":
+            self._cmake.definitions["MACOS"] = True
+        self._cmake.definitions["CRYPTO_LIB"] = self.options.crypto_lib
+        self._cmake.definitions["MSIX_PACK"] = self.options.pack
+        self._cmake.definitions["MSIX_SAMPLES"] = False
+        self._cmake.definitions["MSIX_TESTS"] = False
+        self._cmake.definitions["SKIP_BUNDLES"] = self.options.skip_bundles
+        self._cmake.definitions["USE_EXTERNAL_ZLIB"] = self.options.use_external_zlib
+        self._cmake.definitions["USE_SHARED_ZLIB"] = self.options["zlib"].shared
+        self._cmake.definitions["USE_VALIDATION_PARSER"] = self.options.use_validation_parser
+        self._cmake.definitions["XML_PARSER"] = self.options.xml_parser
+        self._cmake.configure()
+        return self._cmake
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+            self.options.crypto_lib = "crypt32"
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, "14")
+
+    def requirements(self):
+        if self.settings.os == "Linux" and not self.options.skip_bundles:
+            self.requires("icu/69.1")
+        if self.options.crypto_lib == "openssl":
+            self.requires("openssl/1.0.2t")
+        if self.options.use_external_zlib:
+            self.requires("zlib/1.2.11")
+        if self.options.xml_parser == "xerces":
+            self.requires("xerces-c/3.2.3")
+
+    def validate(self):
+        if self.settings.os != "Android" and self.options.xml_parser == "javaxml":
+            raise ConanInvalidConfiguration("javaxml is supported only for Android")
+        if self.settings.os == "Linux" and self.settings.compiler != "clang":
+            raise ConanInvalidConfiguration("Only clang is supported on Linux")
+        # openssl/1.0.2t is not available for Macos+armv8 at the moment
+        if self.settings.os == "Macos" and self.options.crypto_lib == "openssl":
+            raise ConanInvalidConfiguration("openssl is not supported for MacOS at the moment")
```

Comment:
```suggestion
```

This was a missing package on CCI, after #6588 the package should be available.

Reply:
[Removed](https://github.com/conan-io/conan-center-index/pull/6475/commits/1d6d01a088a395ac07835c576cc5b8084e0f5fe3) restriction for `openssl`